<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowGestionnaire</class>
 <widget class="QMainWindow" name="MainWindowGestionnaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidgetGestionnaire">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabPersonnel">
       <attribute name="title">
        <string>Employees</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="10" column="2" rowspan="2" colspan="2">
         <widget class="QLineEdit" name="lineEditLogin"/>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="QTableWidget" name="tableWidgetEmployes">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelNomEmploye_2">
          <property name="text">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditNom"/>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditPrenom"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelPrenomEmploye">
          <property name="text">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButtonSupprimer">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Fire</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="codepostal">
          <item>
           <widget class="QLabel" name="labelCodePostal">
            <property name="text">
             <string>Postal code</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCP"/>
          </item>
         </layout>
        </item>
        <item row="3" column="2" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelVilleEmploye">
            <property name="text">
             <string>City</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditVille"/>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="labelRueEmploye">
          <property name="text">
           <string>Address</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditAdresse"/>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="labelMail">
          <property name="text">
           <string>Mail</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditMail"/>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="labelTypeEmploye">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButtonGestionnaire">
            <property name="text">
             <string>Gestionnaire</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonControleur">
            <property name="text">
             <string>Contrôleur</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="2" rowspan="2" colspan="2">
         <widget class="QLineEdit" name="lineEditTel"/>
        </item>
        <item row="9" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditTelPort"/>
        </item>
        <item row="8" column="0" rowspan="2">
         <widget class="QPushButton" name="pushButtonMaj">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="labelTelEmploye_2">
          <property name="text">
           <string>Cellphone</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="labelLoginEmploye">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item row="10" column="4" colspan="2">
         <widget class="QLabel" name="labelCaracteresExiges">
          <property name="font">
           <font>
            <family>Monospace</family>
            <pointsize>9</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>min.3 caractères </string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" rowspan="2">
         <widget class="QPushButton" name="pushButtonValider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>184</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="12" column="3" colspan="2">
         <widget class="QPushButton" name="pushButtonMDP">
          <property name="text">
           <string>update password</string>
          </property>
         </widget>
        </item>
        <item row="12" column="5">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="labelTelEmploye">
          <property name="text">
           <string>Phone</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVisites">
       <attribute name="title">
        <string>Visits</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabProduits">
       <attribute name="title">
        <string>Products</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1" colspan="4">
           <widget class="QTableWidget" name="tableWidgetProduits">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>validation :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="pushButtonRefuserProduit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>✗</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="pushButtonValiderProduit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>✔</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>nom :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="3">
           <widget class="QLineEdit" name="lineEditProduit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QPushButton" name="pushButtonAddProduit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QPushButton" name="pushButtonDelProduit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Product : </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Categorie :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QTableWidget" name="tableWidgetCategories">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>nom :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditCategorie">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonAddCategorie">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButtonDelCategorie">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Shelf :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QTableWidget" name="tableWidgetRayons">
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>nom :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditRayon"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonAddRayon">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButtonDelRayon">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStatistiques">
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQUit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQUit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tableWidgetEmployes</tabstop>
  <tabstop>pushButtonSupprimer</tabstop>
  <tabstop>pushButtonMaj</tabstop>
  <tabstop>pushButtonValider</tabstop>
  <tabstop>lineEditNom</tabstop>
  <tabstop>lineEditPrenom</tabstop>
  <tabstop>lineEditCP</tabstop>
  <tabstop>lineEditVille</tabstop>
  <tabstop>lineEditAdresse</tabstop>
  <tabstop>lineEditMail</tabstop>
  <tabstop>radioButtonGestionnaire</tabstop>
  <tabstop>radioButtonControleur</tabstop>
  <tabstop>lineEditTel</tabstop>
  <tabstop>lineEditTelPort</tabstop>
  <tabstop>lineEditLogin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
